### DISCUSSION TITLE
Using mswjs with Blazor wasm

### DISCUSSION THREAD

**Discussion by sokrethchhem** [mswjs/msw#1444] [Q&A]
Hello,
How can I integrate mswjs with a Blazor application?

Thanks
Sokreth

**Comment by kettanaito** [comment]
Hi, @sokrethchhem. 

You can integrate MSW into any browser or Node.js process. In the browser, MSW can intercept requests which a Service Worker can intercept. If in doubt, refer to the specification and usage examples of Service Worker + Blazor on the web, it's not specific to this library. In Node.js, it can intercept any request issued by `http`/`https`.
